pkgs/development/libraries/mesa/opencl.patch

```diff
@@ -69,3 +82,16 @@ index 14df6b86f7f..adcd5110342 100644
    )
  
    if meson.version().version_compare('>= 0.58')
+diff --git a/src/gallium/targets/rusticl/meson.build b/src/gallium/targets/rusticl/meson.build
+index a968dee52db..ebda0045ea8 100644
+--- a/src/gallium/targets/rusticl/meson.build
++++ b/src/gallium/targets/rusticl/meson.build
+@@ -59,7 +59,7 @@ configure_file(
+   input : 'rusticl.icd.in',
+   output : 'rusticl.icd',
+   install : true,
+-  install_dir : join_paths(get_option('sysconfdir'), 'OpenCL', 'vendors'),
++  install_dir : join_paths(get_option('prefix'), 'etc', 'OpenCL', 'vendors'),
+ )
+ 
+ if meson.version().version_compare('>= 0.58')
```

Comment:
Maybe consider adding a `sysconfdir_install` option like pulseaudio and fwupd?
https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/linux/firmware/fwupd/add-option-for-installation-sysconfdir.patch
https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/pulseaudio/add-option-for-installation-sysconfdir.patch

Reply:
We should actually probably just not install those files, as we end up deleting them in `postInstall` anyway.